src/McBopomofo.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "McBopomofo.h"
 
+#include <Fcitx5/Module/fcitx-module/notifications/notifications_public.h>
```

Comment:
This seems to be the case, and is beyond my CMake-fu to see if this can be fixed upstream (as fcitx-utils doesn't require spec'ing out the full path like the line here).

Reply:
@lukhnos I believe we can simply use `#include <notifications_public.h>`.

For Fcitx5Modules, the upstream provides https://github.com/fcitx/fcitx5/blob/master/src/lib/fcitx-utils/Fcitx5ModuleTemplate.cmake.in to generate CMake scrips for all components under `fcitx-module`. This is done by invoking the macro `fcitx5_export_module` from the components' CMake scripts.

https://github.com/fcitx/fcitx5/blob/master/src/lib/fcitx-utils/Fcitx5Macros.cmake#L73
https://github.com/fcitx/fcitx5/blob/master/src/modules/notifications/CMakeLists.txt#L11

This macro generates the following CMake script (installed by `fcitx5-modules-dev`):

```cmake
add_library(notifications-interface INTERFACE)                                  
add_library(Fcitx5::Module::Notifications ALIAS notifications-interface)                                                                                                 
set_target_properties(notifications-interface PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "/usr/include/Fcitx5/Module/fcitx-module/notifications")
```

When we tell CMake to link our project against `Fcitx5::Module::Notifications`, it effectively adds `-I/usr/include/Fcitx5/Module/fcitx-module/notifications` to the compiler flags.

https://github.com/openvanilla/fcitx5-mcbopomofo/blob/master/src/CMakeLists.txt#L61

With the `-I/usr/include/Fcitx5/Module/fcitx-module/notifications` flag, we can simply use `#include <notifications_public.h>`.

Reply:
TIL and updated, thank you for checking this!